src/sql/workbench/contrib/editData/browser/editDataInput.ts

```diff
@@ -93,7 +93,7 @@ export class EditDataInput extends EditorInput implements IConnectableInput {
 	// Getters/Setters
 	public get tableName(): string { return this._tableName; }
 	public get schemaName(): string { return this._schemaName; }
-	public get uri(): string { return this._uri.toString(); }
+	public get uri(): string { return unescape(this._uri.toString()); }
```

Comment:
Make sure you check that this URI isn't used anywhere else that expects the escaped URI